workbench_access.module

```diff
@@ -204,35 +203,29 @@ function workbench_access_node_create_access(AccountInterface $account, $context
  */
 function workbench_access_views_data_alter(array &$data) {
   $manager = \Drupal::getContainer()->get('plugin.manager.workbench_access.scheme');
-  try {
-    if ($scheme = $manager->getActiveScheme()) {
-      $data['node']['workbench_access_section'] = array(
-        'title' => t('Workbench Section'),
-        'help' => t('The sections to which this content belongs.'),
-        'field' => array(
-          'id' => 'workbench_access_section',
-        ),
-        'filter' => array(
-          'field' => 'nid',
-          'id' => 'workbench_access_section',
-        ),
-      );
-      $data['users']['workbench_access_section'] = array(
-        'title' => t('Workbench Section'),
-        'help' => t('The sections to which this user belongs.'),
-        'field' => array(
-          'id' => 'workbench_access_user_section',
-        ),
-        'filter' => array(
-          'field' => 'uid',
-          'id' => 'workbench_access_section',
-        ),
-      );
-    }
-  }
-  catch (PluginNotFoundException $e) {
-    // This can occur when a config sync is running that enables the module
-    // that provides the active scheme. No action required.
+  if ($scheme = $manager->getActiveScheme()) {
```

Comment:
I moved catching to the manager->getActiveScheme method - there were at least a couple of other places that we could get a plugin not found exception, so made it central